src/tests/test_utils/functional_test_utils/layer_tests_summary/run_conformance.py

```diff
@@ -70,7 +70,7 @@ def parse_arguments():
     parser.add_argument("-m", "--models_path", help=models_path_help, type=str, required=False,
                         default=NO_MODEL_CONSTANT)
     parser.add_argument("-ov", "--ov_path", help=ov_help, type=str, required=False, default="")
-    parser.add_argument("-j", "--workers", help=workers_help, type=int, required=False, default=os.cpu_count())
+    parser.add_argument("-j", "--workers", help=workers_help, type=int, required=False, default=(os.cpu_count() - 1))
```

Comment:
To make inline with https://github.com/openvinotoolkit/openvino/blob/41036c1e66690f3cdff0b3de4134ce530a61a4ac/src/tests/test_utils/functional_test_utils/layer_tests_summary/run_conformance.py#L52 and make sure the self-hosted runner is not using all the processors and thus not overwhelming the runner.

Reply:
lets use all CPUs, did we face the issues on GHA runners?